cypress/support/dataTestids.js

```diff
@@ -1,165 +1,165 @@
-/*

```

Comment:
Fila må vel også endre navn til stor "i" for at dette skal virke?
`import { dataTestId } from '../../../support/dataTestIds';`

Reply:
Det har jeg gjort, mulig det er github som er kranglete

Reply:
glemt å adde kanskje?

Reply:
Nei, git lokalt sier at det ikke er noe å commite